vignettes/RUVIIIPRPS_TCGA_BrCa_RNAseq.Rmd

```diff
@@ -508,7 +548,14 @@ for(i in categorical.variables){
 }
 ```
 
+## Gene Variable Associations
+In this section, we calculate the association between expression of individual genes and different variables. These results can be used to assess how well each normalisation performs. 
+
+**NOTE to improve other functions based on this part:**: 
+The results here could be used to help with defining negative control genes. So no need to calculate corr later on if we define them here. 
```

Comment:
The results of gene-variable associations could potentially be used to help with defining negative control genes in later stages.

Reply:
That is a good idea. I will incorporate in the package.